cmd/admin-kms-key-create.go

```diff
@@ -0,0 +1,68 @@
+/*
+ * MinIO Client (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/fatih/color"
+	"github.com/minio/cli"
+	"github.com/minio/mc/pkg/probe"
+	"github.com/minio/minio/pkg/console"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+var adminKMSCreateKeyCmd = cli.Command{
+	Name:   "create",
+	Usage:  "creates a new master key at the KMS",
+	Action: mainAdminKMSCreateKey,
+	Before: initBeforeRunningCmd,
+	Flags:  globalFlags,
+	CustomHelpTemplate: `NAME:
+  {{.HelpName}} - {{.Usage}}
+
+USAGE:
+  {{.HelpName}} TARGET [KEY_NAME]
```

Comment:
[KEY_NAME] indicates that its optional, do we autogenerate key if not provided? if not it should be 

```suggestion
  {{.HelpName}} TARGET KEYNAME
```